module-assets/.pre-commit-config.yaml

```
@@ -185,3 +185,10 @@ repos:
   rev: v0.1.29
   hooks:
     - id: helmlint
+# ibm_catalog.json lint
+- repo: https://github.com/python-jsonschema/check-jsonschema
+  rev: 0.33.0
+  hooks:
+    - id: check-jsonschema
```

Suggest to update the ID so its clear what the hook is doing
```suggestion
    - id: Validate ibm_catalog.json schema
```